plasma_framework/migrations/config.js

```diff
@@ -11,6 +11,11 @@ const development = {
             mvp: 1,
             moreVp: 2,
         },
+        // defines how much gas should be consider safe when transferring ETH
+        // With version control to record old gas stipend while be able to adjust to ETH network change for upgrade.
```

Comment:
```suggestion
        // Under version control so that it can be upgraded to reflect Ethereum network changes, while keeping a record of the previous gas stipend.
```